benches/bench.rs

```diff
@@ -2294,13 +2211,11 @@ fn bench_create_map(c: &mut Criterion) {
     let mut group = c.benchmark_group(function.to_string());
 
     for input_size in &INPUT_SIZES {
-        // want a warmed-up marf
```

Comment:
Do we preserve comments of this type (doesn't have to be exactly /`/ want a warmed-up marf`) - could be helpful to denote we are doing something "different" in some functions for a reason

Reply:
we are really only ever doing 1 of 3 things now:

1. Using a fresh MemoryBackingStore
2. ^ + initializing a SimHeadersDB
3. Passing in a MemoryBackingStore with preloaded state

I think 1 and 2 are pretty self explanatory, and 3 only happens twice I think.